docs/examples/users.md

```diff
@@ -456,15 +456,16 @@ Here's a function to get the email of the currently authenticated
 user.
 
 ```sql
+-- Prevent current_setting('postgrest.claims.email') from raising
+-- an exception if the setting is not present. Default it to ''.
+ALTER DATABASE :database_name SET postgrest.claims.email TO '';
```

Comment:
Maybe it's better to use a fictitious database name instead of using a variable `:database_name`.